app.rb

```diff
@@ -0,0 +1,161 @@
+require_relative 'book'
+require_relative 'person'
+require_relative 'rental'
+require_relative 'student'
+require_relative 'teacher'
+
+class App
+  MENU_OPTIONS = {
+    '1' => :list_books,
+    '2' => :list_people,
+    '3' => :create_person,
+    '4' => :create_book,
+    '5' => :create_rental,
+    '6' => :list_all_rentals,
+    '7' => :exit_program
+  }.freeze
```

Comment:
- [ ] While this was not highlighted by the previous reviewers, kindly note that this was supposed to be included inside main.rb and not really here in app.rb. Please refer to the specific requirement ias provided in the project requirements: **This is an [OPTIONAL] but highly recommend to make**

![image](https://github.com/axcell-0/school-library/assets/86318284/eabf5314-5620-4f42-a804-2c5f8739d2fa)